src/realm/sync/client.cpp

```diff
@@ -1559,13 +1459,6 @@ void SessionWrapper::on_download_completion()
         m_upload_completion_handlers.push_back(std::move(handler)); // Throws
         m_sync_completion_handlers.pop_back();
     }
-
-    if (m_flx_subscription_store && m_flx_pending_mark_version != SubscriptionSet::EmptyVersion) {
-        m_sess->logger.debug("Marking query version %1 as complete after receiving MARK message",
-                             m_flx_pending_mark_version);
```

Comment:
It would be nice to keep this debug message to show when the bootstrap/version is complete. Perhaps move it to be printed out in `download_complete()`.